include/dev/RedundantADC.hpp

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include <EVT/io/ADC.hpp>
+
+namespace IO = EVT::core::IO;
+
+namespace RedundantADC {
```

Comment:
You shouldn't have a namespace for every class
```suggestion
namespace HIB::DEV {
```